server/server.go

```diff
@@ -46,6 +47,13 @@ var _ v1alpha1.CSIDriverProviderServer = &Server{}
 
 // Mount implements provider csi-provider method
 func (s *Server) Mount(ctx context.Context, req *v1alpha1.MountRequest) (*v1alpha1.MountResponse, error) {
+	deadline, ok := ctx.Deadline()
+	if !ok {
+		log.Printf("Mount() called without a deadline.")
```

Comment:
Should we use a default deadline if it was called without one?

Reply:
I dont think it matters much, as the SM client library should set default deadlines based on the SM service configuration.